lib/Backends/CPU/CMakeLists.txt

```diff
@@ -13,52 +13,58 @@ set(CMAKE_LLIR_CREATE_SHARED_LIBRARY "${LLVM_LINK_BIN} -o <TARGET> <OBJECTS>")
 set(CMAKE_LLIR_CREATE_SHARED_MODULE "${LLVM_LINK_BIN} -o <TARGET> <OBJECTS>")
 set(CMAKE_LLIR_CREATE_STATIC_LIBRARY "${LLVM_LINK_BIN} -o <TARGET> <OBJECTS>")
 
-add_library(CPURuntime
-              libjit/libjit.cpp
-              libjit/libjit_conv.cpp
-              libjit/libjit_matmul.cpp)
+set(CPURunttimeCompileOptions 
+      -ffast-math
+      -fno-finite-math-only
+      -g0
+      -emit-llvm 
+      -O0)
+
+set(CPURuntime_val CPURuntime)
+  
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${CPURuntime_val})
+file(GLOB_RECURSE libjit_SOURCES LIST_DIRECTORIES false libjit/*.cpp)
```

Comment:
Please don't do the glob.  It causes issues with dependency tracking, and you don't know necessarily what files are going to get included.

Reply:
Elsewhere we explicitly list the .cpp files included by a library.  As annoying as this is, as @compnerd mentions, using glob() causes issues with dependency tracking since the build system may not know when files are added.  (Some build systems like buck solve this by using a filesystem watcher - buck uses watchman - but cmake doesn't.)